common/utils.js

```diff
@@ -20,6 +20,13 @@
         "*": "/record"
     })
 
+    .constant("appNames", {
```

Comment:
Is this constant necessary? I don't see it being used anywhere. 

We have this set enumerated already in `ermrestJS` attached to the module component based on their annotation tags.